packages/wrangler/src/api/startDevWorker/utils.ts

```diff
@@ -41,3 +44,158 @@ export function urlFromParts(
 
 	return url;
 }
+
+export function unwrapHook<T extends string | number | object>(
+	hook: Hook<T>
+): T | Promise<T>;
+export function unwrapHook<T extends string | number | object>(
+	hook: Hook<T> | undefined
+): T | Promise<T> | undefined;
+export function unwrapHook<T extends string | number | object>(hook: Hook<T>) {
+	return typeof hook === "function" ? hook() : hook;
+}
+
+export class MissingConfigError extends Error {
+	constructor(key: string) {
+		super(`Missing config value for ${key}`);
+	}
+}
+
+export async function getBinaryFileContents(file: File<string | Uint8Array>) {
+	if ("contents" in file) {
+		if (file.contents instanceof Buffer) {
+			return file.contents;
+		}
+		return Buffer.from(file.contents);
+	}
+	return readFile(file.path);
```

Comment:
do we need to specify the encoding here?

Reply:
No, because we're returning a `Buffer` rather than a string